### PR TITLE
fix: use /dist/ as default publicPath for webpack

### PR DESCRIPTION
**Pull request by wejendorp**
Related to https://github.com/kapetacom/sdk-nodejs-server/pull/17 - we should default to use `/dist` to serve assets.

### CODE REVIEW

**Review by hofmeister — COMMENTED** [review]
You sure about this? PublicPath is for web isnt it?

**Comment by wejendorp** [comment]
Yea it's for web. The idea is to always serve assets on `/dist` prefix. That way we can use a static middleware without fallthrough. I believe it should be the same as whatever path we mount the output folder on.

**Comment by hofmeister** [comment]
A few things:
1.  I think this will affect the way fragments gets served - so you'll need to handle that. I believe it cant be a absolute path for one
2. I'd give it a better name than "dist". For cosmetic reasons I think people would dislike that. I would at least. maybe app/assets, assets/, assets/<name>/ or similar

File: templates/kapeta/block-type-frontend/webpack.development.config.js.hbs [hunk 1]
```diff
@@ -51,7 +50,7 @@ const config = {
     output: {
         path: Path.join(__dirname, "dist"),
         filename: "[name].[contenthash].bundle.js",
-        publicPath: "",
+        publicPath: "/dist/",
```
Comment (andertun) [inline templates/kapeta/block-type-frontend/webpack.development.config.js.hbs:53]: I found out in `web-metrics` that without doing this change the app couldn't run in prod mode. I saw this error in the console:

```
No renderPage method on response object. Using fallback render.
````

And in the returned html I saw this error:

```
window.BACKEND_ERROR = {"errno":-2,"code":"ENOENT","syscall":"stat","path":"/Users/harald/code/kapeta/web-metrics/dist/metrics","expose":false,"statusCode":404,"status":404};
```

Changing the publicPath to `"/dist/"` solved it. So the current latest SDK's require this change to work.

**Pull request merged**